### PR TITLE
0058-feel-number-function

### PR DESCRIPTION
**Pull request by StrayAlien**

### CODE REVIEW

**Comment by agilepro** [comment]
This passes in Drools.   Should we merge?

**Comment by opatrascoiu** [comment]
DMN 1.2 specifies the set of possible values for the arguments (e.g. ".", "," and null) in Table 66. We should probably add a test for values outside the allowed domain. For example, `number("123 .00", "abc", 123)`. 

Also, `number("1,023.00", ".", ",")` to test a common error when the arguments are switched by accident.

File: TestCases/compliance-level-3/0058-feel-number-function/0058-feel-number-function.dmn [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+        <allowedAnswers>1000000.01 (number)</allowedAnswers>
+        <variable typeRef="number" name="decision002"/>
+        <literalExpression>
+            <text>number("1,000,000.01", ",", ".")</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision003" id="_decision003">
+        <description>Tests FEEL expression: 'number("1 000 000,01", " ", "."))' and expects result: '1000000.1 (number)'</description>
+        <question>Result of FEEL expression 'number("1 000 000,01", " ", "."))'?</question>
+        <allowedAnswers>1000000.1 (number)</allowedAnswers>
```
Comment (tarilabs) [inline TestCases/compliance-level-3/0058-feel-number-function/0058-feel-number-function.dmn]: minor typo, I believe is meant to be `.01`

File: TestCases/compliance-level-3/0058-feel-number-function/0058-feel-number-function.dmn [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+        <allowedAnswers>1000000.01 (number)</allowedAnswers>
+        <variable typeRef="number" name="decision005"/>
+        <literalExpression>
+            <text>number("1000000.01", null, ".")</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision006" id="_decision006">
+        <description>Tests FEEL expression: 'number("1,000,000.01", null, "."))' and expects result: '1000000.00 (number)'</description>
+        <question>Result of FEEL expression 'number("1,000,000.01", null, "."))'?</question>
+        <allowedAnswers>1000000.00 (number)</allowedAnswers>
```
Comment (tarilabs) [inline TestCases/compliance-level-3/0058-feel-number-function/0058-feel-number-function.dmn]: minor typo, I believe is meant to be `.01` here

Comment (StrayAlien) [reply TestCases/compliance-level-3/0058-feel-number-function/0058-feel-number-function.dmn]: As Homer would say. Doh!!

Comment (tarilabs) [reply TestCases/compliance-level-3/0058-feel-number-function/0058-feel-number-function.dmn]: no worries; these as mentioned are only minor typos, but personally I prefer to review stuff --also manually-- before merge ;)

**Comment by agilepro** [comment]
Please address the typos and then we will merge it.

**Comment by StrayAlien** [comment]
@agilepro typos fixed.

**Comment by StrayAlien** [comment]
@agilepro ... yet another typo fix .... apols.

**Pull request merged**